bundles/org.palladiosimulator.dataflow.confidentiality.analysis/src/org/palladiosimulator/dataflow/confidentiality/analysis/sequence/entity/AbstractActionSequenceElement.java

```diff
@@ -36,7 +37,7 @@ public AbstractActionSequenceElement(List<DataFlowVariable> dataFlowVariables, L
      * @param variables List of {@link DataFlowVariable}s propagated from the precursor
      * @return Returns a new Sequence element with the updated Node- and DataFlowVariables
      */
-    public abstract AbstractActionSequenceElement<T> evaluateDataFlow(List<DataFlowVariable> variables);
+    public abstract AbstractActionSequenceElement<T> evaluateDataFlow(List<DataFlowVariable> variables, PCMResourceLoader resourceLoader);
```

Comment:
Here, the non-PCM-specific super class gets a PCM dependency which violates the dependency inversion principle. This is OK for now but we should have look at it in the upcoming refactoring #51.